cmd/get/execution.go

```diff
@@ -74,13 +99,11 @@ func getExecutionFunc(ctx context.Context, args []string, cmdCtx cmdCore.Command
 		}
 		executions = append(executions, execution)
 	} else {
-		executionList, err := cmdCtx.AdminClient().ListExecutions(ctx, &admin.ResourceListRequest{
-			Limit: 100,
-			Id: &admin.NamedEntityIdentifier{
-				Project: config.GetConfig().Project,
-				Domain:  config.GetConfig().Domain,
-			},
-		})
+		transformFilters,err := filters.BuildResourceListRequestWithName(executionConfig.Filter, "")
+		if err != nil {
+			return err
+		}
+		executionList, err := cmdCtx.AdminClient().ListExecutions(ctx, transformFilters)
```

Comment:
Ideally we dont want direct adminClient calls from this layer and similar to other fetcher calls in launchplan,task and workflow, that interface should be the link between commandline flytectl interface and adminClient calls interface

Reply:
can we do in a separate pr because we are increasing the scope of the pr?